### ISSUE TITLE
Redesign of the edit dialog of personal records

### ISSUE THREAD

**Issue by Serg-Norseman** [Serg-Norseman/GEDKeeper#88]
Source: https://groups.google.com/forum/#!topic/gedkeeper-ru/eq1uKbCngkU and many other

**Comment by fire-eggs** [comment]
I tried mocking up some new variants of the "Edit individual" dialog, using Pencil. I'm not too happy with the results; I might try again using the visual studio designer. [Pencil is a little limited; other wireframe tools too complex or pricey...]

 In the meantime, this is a conversation starter. Mainly my own opinions, consider with skepticism:

[Indi mock.zip](https://github.com/Serg-Norseman/GEDKeeper/files/1569248/Indi.mock.zip)

I did a quick survey of a few genealogy tools for ideas:
[IndiDialogSurvey.zip](https://github.com/Serg-Norseman/GEDKeeper/files/1569249/IndiDialogSurvey.zip)
